web/portal/brand/src/entities/CompanyBalances/CompanyBalances.tsx

```diff
@@ -4,6 +4,14 @@ import _ from '@irontec/ivoz-ui/services/translations/translate';
 
 const CompanyBalances = {
   ...Company,
+  acl: {
+    read: true,
+    create: false,
+    update: false,
+    detail: false,
+    delete: false,
+    iden: 'Prepaid Balance',
```

Comment:
This is used check out if a restricted admin can access this entity. It must be a valid ACL iden (fetched when you call /my/profile). So:
```suggestion
    iden: 'Companies',
```